Api.dockerfile

```diff
@@ -0,0 +1,32 @@
+# ** Build
+
+FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS build
+WORKDIR /tmp/pwdls/
+
+COPY Directory.Build.props ./
+COPY src/Common src/Common
+COPY src/Service src/Service
+COPY src/Api src/Api
+
+RUN dotnet publish src/Api/ \
+    --configuration Release \
+    --self-contained \
+    --use-current-runtime \
+    --output src/Api/bin/publish/
+
+# ** Run
+
+# Use `runtime-deps` instead of `runtime` because we have a self-contained assembly
+FROM mcr.microsoft.com/dotnet/runtime-deps:7.0-alpine AS run
+WORKDIR /opt/pwdls/
+
+EXPOSE 80
+EXPOSE 443
+
+# Alpine image doesn't come with the ICU libraries pre-installed, so we need to install them manually.
+# Technically, we shouldn't need globalization support in the API, but some EF queries fail without it at the moment.
+RUN apk add --no-cache icu-libs
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
```

Comment:
This is something we may consider looking into in the future. It would make the image even smaller and will probably speed up the app (in production too) as well.